e2e/config/config.go

```diff
@@ -132,6 +145,8 @@ type Config struct {
 	TestEnvConfig testenv.Config `mapstructure:",squash"`
 	// TestSuiteParams contains test parameters for each environment.
 	TestSuiteParams *TestSuiteParams
+	// BenchmarkConfig contains the configuration for benchmarking.
+	BenchmarkConfig benchmark.Config `mapstructure:",squash"`
```

Comment:
lets not squash here to make it clear and consistent
```suggestion
	BenchmarkConfig benchmark.Config `mapstructure:"benchmark"`
```